src/app/workflow-editor/graph-editor/graph-editor/workflow-graph-editor.component.scss

```diff
@@ -8,7 +8,17 @@ svg.cwl-workflow {
     height: 100%;
     position: relative;
 
+}
+
+.svg-btns.btn-group {
+    bottom: 30px;
+    float: right;
+    position: relative;
+    right: 10px;
 
+    >.btn:nth-of-type(4), >.btn:nth-of-type(5) {
```

Comment:
Avoid ordering-specific selectors. If we add more buttons, this styling will break. Instead, use `.btn-group` around just the buttons that need to be grouped.

Also avoid direct descendent selectors (`>`)